### PR TITLE
feat: [PIE-5371]: run pipeline form support for multi svc

### PR DESCRIPTION
**Pull request by vkbansal-harness**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by diptiman-adak** [comment]
Quality gate Quality gate: **_Failed_** 
 


            
 #### 7 Issues 
* Code issue 7 Code issues 

 [Details](https://app.embold.io/organization/gh/harness/repositories/e864badd0b95ab81c060b15b85f15701/pull-request/11243)

**Comment by vkbansal-harness** [comment]
retrigger titlecheck

**Comment by vkbansal-harness** [comment]
retrigger jest

**Comment by vkbansal-harness** [comment]
trigger cypress

File: src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx [hunk 1]
```diff
@@ -847,12 +846,13 @@ export function StageInputSetFormInternal({
                 customStepProps={{
                   stageIdentifier,
                   deploymentType: deploymentStage?.deploymentType,
-                  gitOpsEnabled: deploymentStage?.gitOpsEnabled
+                  gitOpsEnabled: deploymentStage?.gitOpsEnabled,
+                  allValues: pick(deploymentStage, ['service', 'services'])
```
Comment (AshwinBhatkal-Harness) [inline src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx:850]: Should we remove services from here? We are going to load this component if the template contains service, so services wouldn't be there

**Review by AshwinBhatkal-Harness — APPROVED** [review]

File: src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx (hunk 1 above)
Comment (vkbansal-harness) [reply src/modules/70-pipeline/components/PipelineInputSetForm/StageInputSetForm.tsx:850]: I will be adding that support in next PR

**Review by devinder-harness — APPROVED** [review]

**Review by YounixIO — APPROVED** [review]

**Pull request merged**